fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -841,7 +841,10 @@ ACTOR Future<Void> refreshReadIteratorPool(std::shared_ptr<ReadIteratorPool> rea
 				when(wait(delay(SERVER_KNOBS->ROCKSDB_READ_RANGE_ITERATOR_REFRESH_TIME))) {
 					readIterPool->refreshIterators();
 				}
-				when(waitNext(readIterPool->getDeleteIteratorsFuture())) { readIterPool->refreshIterators(); }
+				when(waitNext(readIterPool->getDeleteIteratorsFuture())) {
+					wait(delay(0.01));
```

Comment:
@jzhou77 Can this wait make the call asynchronous?

Reply:
Yes. I think you can just use `wait(delay(0.0));` here as well. Please also add a comment like "add a delay(0) to ensure the rest code of the caller runs before refreshing iterators, i.e., making the `refreshIterators()` callback here asynchronous".